src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -412,10 +412,9 @@ public async Task Revisions()
                 await service.SetAsync(testSettingUpdate);
 
                 // Test
-                var filter = new BatchRequestOptions();
-                filter.Key = setting.Key;
-                filter.Revision = DateTimeOffset.MaxValue;
-                SettingBatch batch = await service.GetRevisionsAsync(filter, CancellationToken.None);
+                var selector = new ConfigurationSelector(setting.Key);
```

Comment:
ConfigurationSelector sounds like it selects a configuration. But it selects a setting. Would SettingSelector be a better name?

Reply:
FYI @conniey 

SettingSelector sounds good. I like that doesn't include batch like before to mak eit more generic